packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -89,26 +93,18 @@ class SimpleFluidError implements IFluidErrorBase {
         this.message = errorProps.message;
         this.stack = errorProps.stack;
         this.name = errorProps.name;
+
+        //* PR: Do I need to avoid adding optional ones, or is it ok to add an explicit undefined value?
+        // (like I wonder if it will show up in Kusto as "undefined" -- Oh I think not, recalling the Bohemia code)
+        this.addTelemetryProperties(errorProps);
```

Comment:
undefined should be fine. should just sterilize without the property as long as it isn't stringified is a custom way, which i don't think it will be.

Reply:
um why is this comment still here, ha.